docs/types/reference-types.rst

```diff
@@ -85,8 +85,10 @@ Data locations are not only relevant for persistency of data, but also for the s
             // The following does not work; it would need to create a new temporary /
             // unnamed array in storage, but storage is "statically" allocated:
             // y = memoryArray;
-            // This does not work either, since it would "reset" the pointer, but there
-            // is no sensible location it could point to.
+            // On the other hand: "delete y" is not valid, as assignments to local variables
+            // referencing storage objects can only be made from existing storage objects.
+            // It would "reset" the pointer, but there is no sensible location it could point to.
+            // See "delete" under Operators
```

Comment:
"For more details see the documentation of the "delete" operator"?

Reply:
Roughtly equivalent. I was trying to point the reader to the literal section called "delete" in the Operators section of docs, which is the one where I pulled the comment from.